tensorflow/compiler/xla/service/gpu/gpu_compiler.cc

```diff
@@ -343,11 +343,14 @@ Status GpuCompiler::PrepareHloModuleForIrEmitting(HloModule* hlo_module) {
 // TODO(cheshire): Duplication with gpu_conv_algorithm picker, figure out a
 // right way to share this.
 static bool RequireDeterminism() {
-  bool deterministic_ops = false;
-  TF_CHECK_OK(tensorflow::ReadBoolFromEnvVar("TF_DETERMINISTIC_OPS",
-                                             /*default_val=*/false,
-                                             &deterministic_ops));
-  return deterministic_ops;
+  static bool require_determinism = [] {
```

Comment:
Is it too much to ask to extract a common function with `gpu_conv_algorithm_picker` while we're at it?

Reply:
They have functionality that is different enough that it might not be worth trying to find the common factor, especially since they're intended to be temporary. If they were combined and placed in another source file, does one that already exists stand out to you as a good candidate, or would a new one make more sense?